guides/doc-Release_Notes/topics/foreman.adoc

```diff
@@ -9,15 +9,7 @@ There are no highlights with Foreman {ProjectVersion}.
 [id="foreman-upgrade-warnings"]
 == Upgrade Warnings
 
-// If this section would be empty otherwise, uncomment the following line:
-// There are no upgrade warnings with Foreman {ProjectVersion}.
-
-=== PostgreSQL upgraded to 13 on Enterprise Linux 8
-
-On Enterprise Linux 8 PostgreSQL is upgraded from 12 to 13, to match the version used on Enterprise Linux 9.
-Users are required to change the DNF module, but the actual upgrade, including data migration, happens when the installer is executed for the first time.
-During the upgrade a backup of the data is created in `/var/lib/pgsql/data-old`.
-This backup can be removed once the upgrade is completed.
+There are no upgrade warnings with Foreman {ProjectVersion}.
```

Comment:
```suggestion
// If this section would be empty otherwise, uncomment the following line:
There are no upgrade warnings with Foreman {ProjectVersion}.
```